src/Components/Aspire.Azure.Storage.Queues/AzureStorageQueueSettings.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Data.Common;
+using Aspire.Azure.Common;
+
+namespace Aspire.Azure.Storage.Queues;
+
+/// <summary>
+/// Provides the client configuration settings for connecting to an Azure Storage queue.
+/// </summary>
+public sealed partial class AzureStorageQueueSettings : AzureStorageQueuesSettings, IConnectionStringSettings
+{
+    /// <summary>
+    ///  Gets or sets the name of the blob container.
+    /// </summary>
+    public string? QueueName { get; set; }
+
+    void IConnectionStringSettings.ParseConnectionString(string? connectionString)
+    {
+        if (string.IsNullOrEmpty(connectionString))
+        {
+            return;
+        }
+
+        var builder = new DbConnectionStringBuilder
+        {
+            ConnectionString = connectionString
+        };
+
+        if (builder.TryGetValue("QueueName", out var containerName))
+        {
+            var stableBuilder = new StableConnectionStringBuilder(connectionString);
```

Comment:
1. Why does this method look so different from `AzureBlobStorageContainerSettings`' `ParseConnectionString`?
2. Why are we using 2 different connection string builders here?